ytmusicapi/auth/oauth/models.py

```diff
@@ -0,0 +1,29 @@
+# stand-in classes for clarity until pydantic implementation
+
+from typing import Union, Literal, TypedDict
+
+DefaultScope = Literal['https://www.googleapis.com/auth/youtube']
+Bearer = Literal['Bearer']
+
+
+class BaseTokenDict(TypedDict):
+    """ Limited token. Does not provide a refresh token. Commonly obtained via a token refresh. """
+    access_token: str
+    expires_in: int
+    scope: Union[str, DefaultScope]
+    token_type: Union[str, Bearer]
+
+
+class FullTokenDict(BaseTokenDict):
```

Comment:
Not really a great name tbh. Better to follow the naming of `RefreshingToken`? So `RefreshingTokenDict`?